pax/plugins/io/AmazonAWS.py

```diff
@@ -0,0 +1,47 @@
+"""Write to Amazon dynamoDB
+"""
+from pax import datastructure, plugin
+import boto3
+
+class WriteDynamoDB(plugin.OutputPlugin):
+    do_input_check = False
+    do_output_check = False
+
+    def startup(self):
+
+        self.dynamodb = boto3.resource('dynamodb')
+        # Create the DynamoDB table.
+        self.table = self.dynamodb.create_table(TableName='processed',
+             KeySchema=[
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->